ex2_system/source/system_tasks.c

```diff
@@ -40,7 +41,8 @@ SAT_returnState start_system_tasks(void) {
       start_diagnostic_daemon() != SATR_OK ||
       start_housekeeping_daemon() != SATR_OK ||
       start_system_stats_daemon() != SATR_OK ||
-      start_state_daemon() != SATR_OK) {
+      start_state_daemon() != SATR_OK ||
+      start_general_service() != SATR_OK) {

```

Comment:
This is for system tasks. Services should be added [here](https://github.com/AlbertaSat/ex2_services/blob/master/Services/source/services.c#L49).